<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RasterVision</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1,7">
      <item>
       <widget class="QTreeView" name="treeView"/>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ZoomableGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,4,4,4">
      <item>
       <widget class="QLabel" name="label_crs">
        <property name="text">
         <string>导入文件以查看坐标系统信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string> </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string> </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="zoom_label">
        <property name="text">
         <string> </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>图像拉伸</string>
     </property>
     <addaction name="action_stretch_linear"/>
     <addaction name="action_stretch_linear_pct1"/>
     <addaction name="action_stretch_linear_pct2"/>
     <addaction name="action_stretch_linear_pct5"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="action_pixelinfo"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>文件</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_pixelinfo">
   <property name="text">
    <string>鼠标取值</string>
   </property>
  </action>
  <action name="action_stretch_linear">
   <property name="text">
    <string>线性</string>
   </property>
  </action>
  <action name="action_stretch_linear_pct1">
   <property name="text">
    <string>线性1%</string>
   </property>
  </action>
  <action name="action_stretch_linear_pct2">
   <property name="text">
    <string>线性2%</string>
   </property>
  </action>
  <action name="action_stretch_linear_pct5">
   <property name="text">
    <string>线性5%</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomableGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>frontend.views.ZoomableGraphicsView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
